ci-operator/step-registry/redhat-performance/benchmark-runner/redhat-performance-benchmark-runner-commands.sh

```
@@ -0,0 +1,199 @@
+  if [[ -s /secret/bastion_address ]] && [[ -s /secret/jh_priv_ssh_key ]]; then
+    BASTION_ADDRESS=$(<"/secret/bastion_address")
+    BASTION_ADDRESS="${BASTION_ADDRESS%$'\n'}"
+    cp /secret/jh_priv_ssh_key /tmp/bastion_key
+    chmod 600 /tmp/bastion_key
+    if ! curl -sk --max-time 5 "${CLUSTER_SERVER}/version" &>/dev/null; then
+      echo "API not directly reachable, starting SOCKS proxy through ${BASTION_ADDRESS}"
+      SSH_ARGS="-i /tmp/bastion_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+      SOCKS_PORT=$((RANDOM % 55536 + 10000))
+      ES_PORT=$(<"/secret/elasticsearch_port")
+      ES_PORT="${ES_PORT%$'\n'}"
+      ssh ${SSH_ARGS} root@"${BASTION_ADDRESS}" -fNT -D "${SOCKS_PORT}" -L "127.0.0.1:${ES_PORT}:localhost:${ES_PORT}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)